scripts/download_test_data.py

```diff
@@ -26,7 +26,7 @@
 import tarfile
 import sys
 
-TEST_DATA_VERSION = "0.3"
+TEST_DATA_VERSION = "0.5"
```

Comment:
Why don't you merge this commit into the one titled "Add test on multi part embedded file. "?